tools/Environments/DevHome.Environments/Strings/en-us/Resources.resw

```diff
@@ -111,15 +111,15 @@
     <comment>Text for terminate button in the context menu that will turn off the virtual machine. This button turns off the power to the virtual machine without gracefully shutting it down first.</comment>
   </data>
   <data name="SortByName.Content" xml:space="preserve">
-    <value>Name</value>
-    <comment>Text for 1st option for sort by drop down box</comment>
+    <value>Name: A-Z</value>
+    <comment>Text for sorting in ascending order of name</comment>
   </data>
   <data name="SortByAltName.Content" xml:space="preserve">
-    <value>Alternative Name</value>
-    <comment>Text for 2nd option for sort by drop down box</comment>
+    <value>Alternative Name: A-Z</value>
+    <comment>Text for sorting by alternative name</comment>
   </data>
   <data name="SortByTextBlock.Text" xml:space="preserve">
-    <value>Sort By</value>
+    <value>Sort:</value>

```

Comment:
Because you have this label separate from being the "Header" of the ComboBox, you're eventually going to have an accessibility bug that the ComboBox doesn't have an accessible name. Consider adding it before you do. (Probably easier to do that than re-template the ComboBox to move the Header to the side)

Reply:
I didn't quite get the suggestion. Could you elaborate? Should we have 'Sort By'?

Reply:
Sorry, this was vague out of context, it's just fresh in my mind since I recently fixed #2319. UIA uses the Header property on a ComboBox for its AccessibleName. If you don't have a Header, you'll have to specify an accessible name. I think the easiest way to do it is adding this to your resw:
```
  <data name="SortSelectionComboBox.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
    <value>Sort by</value>
    <comment>Name of ComboBox where user can choose how to sort</comment>
  </data>
```

You can look at the Environments page in AccessibilityInsights for more context.

Reply:
Thanks for this. Wouldn't have been able to figure this out on my own.